certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/unit/service/GetUnitStatisticsDomainService.java

```diff
@@ -0,0 +1,26 @@
+package se.inera.intyg.certificateservice.domain.unit.service;
+
+import static se.inera.intyg.certificateservice.domain.common.model.Role.CARE_ADMIN;
+
+import java.util.List;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import se.inera.intyg.certificateservice.domain.action.certificate.model.ActionEvaluation;
+import se.inera.intyg.certificateservice.domain.certificate.repository.StatisticsRepository;
+import se.inera.intyg.certificateservice.domain.unit.model.UnitStatistics;
+
+@RequiredArgsConstructor
+public class GetUnitStatisticsDomainService {
+
+  private final StatisticsRepository statisticsRepository;
+
+  public Map<String, UnitStatistics> get(ActionEvaluation actionEvaluation, List<String> unitIds) {
```

Comment:
Maybe use HsaId instead of String in the domain? Same comment as before on the naming of the argument. unitIds -> issuedUnitIds or similar.